stdpopsim/cli.py

```diff
@@ -722,6 +735,11 @@ def run_simulation(args):
                 annot = species.get_annotations(args.dfe_annotation)
                 intervals = annot.get_chromosome_annotations(args.chromosome)
                 intervals_summary_str = f"{annot.id} elements on {args.chromosome}"
+            elif args.dfe_bed_file is not None:
```

Comment:
we're going to want to do some testing to make sure this option isn't called with `--dfe-interval` or `--dfe-annotation`

Reply:
Oh I just saw this now, yes I can do that!

Reply:
added